diff --git a/TaskFlow.Api/Controllers/ActivityController.cs b/TaskFlow.Api/Controllers/ActivityController.cs
index b8c5377..3e27c18 100644
--- a/TaskFlow.Api/Controllers/ActivityController.cs
+++ b/TaskFlow.Api/Controllers/ActivityController.cs
@@ -3,6 +3,7 @@ using TaskFlow.Api.DTOs;
 using TaskFlow.Application.Interfaces;
 using TaskFlow.Domain.Builders;
 using TaskFlow.Domain.Models;
+using Task = System.Threading.Tasks.Task;
 
 namespace TaskFlow.Api.Controllers
 {
diff --git a/TaskFlow.Application/Decorators/LoggingActivityServiceDecorator.cs b/TaskFlow.Application/Decorators/LoggingActivityServiceDecorator.cs
index 91f433c..1368416 100644
--- a/TaskFlow.Application/Decorators/LoggingActivityServiceDecorator.cs
+++ b/TaskFlow.Application/Decorators/LoggingActivityServiceDecorator.cs
@@ -1,5 +1,6 @@
 using TaskFlow.Application.Interfaces;
 using TaskFlow.Domain.Models;
+using Task = System.Threading.Tasks.Task;
 
 namespace TaskFlow.Application.Decorators
 {
diff --git a/TaskFlow.Application/Interfaces/IActivityService.cs b/TaskFlow.Application/Interfaces/IActivityService.cs
index 0ca919f..898e50a 100644
--- a/TaskFlow.Application/Interfaces/IActivityService.cs
+++ b/TaskFlow.Application/Interfaces/IActivityService.cs
@@ -1,4 +1,5 @@
 using TaskFlow.Domain.Models;
+using Task = System.Threading.Tasks.Task;
 
 namespace TaskFlow.Application.Interfaces
 {
diff --git a/TaskFlow.Application/Observers/ITaskObserver.cs b/TaskFlow.Application/Observers/ITaskObserver.cs
index 22de011..297ea4d 100644
--- a/TaskFlow.Application/Observers/ITaskObserver.cs
+++ b/TaskFlow.Application/Observers/ITaskObserver.cs
@@ -1,4 +1,5 @@
 using TaskFlow.Domain.Models;
+using Task = System.Threading.Tasks.Task;
 
 namespace TaskFlow.Application.Observers
 {
diff --git a/TaskFlow.Application/Services/ActivityService.cs b/TaskFlow.Application/Services/ActivityService.cs
index df665c4..1d744de 100644
--- a/TaskFlow.Application/Services/ActivityService.cs
+++ b/TaskFlow.Application/Services/ActivityService.cs
@@ -2,6 +2,7 @@ using TaskFlow.Application.Interfaces;
 using TaskFlow.Application.Strategies;
 using TaskFlow.Domain.Models;
 using TaskFlow.Infrastructure.Interfaces;
+using Task = System.Threading.Tasks.Task;
 
 namespace TaskFlow.Application.Services
 {
diff --git a/TaskFlow.Infrastructure/Repositories/ActivityRepository.cs b/TaskFlow.Infrastructure/Repositories/ActivityRepository.cs
index f82714f..fb22d9c 100644
--- a/TaskFlow.Infrastructure/Repositories/ActivityRepository.cs
+++ b/TaskFlow.Infrastructure/Repositories/ActivityRepository.cs
@@ -2,6 +2,7 @@ using Microsoft.EntityFrameworkCore;
 using TaskFlow.Domain.Models;
 using TaskFlow.Infrastructure.Data;
 using TaskFlow.Infrastructure.Interfaces;
+using Task = System.Threading.Tasks.Task;
 
 namespace TaskFlow.Infrastructure.Repositories
 {
